alkira/resource_alkira_peering_gateway_cxp.go

```diff
@@ -0,0 +1,193 @@
+package alkira
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAlkiraPeeringGatewayCxp() *schema.Resource {
+	return &schema.Resource{
+		Description:   "Manage CXP Peering Gateways.",
+		CreateContext: resourceAlkiraPeeringGatewayCxpCreate,
+		ReadContext:   resourceAlkiraPeeringGatewayCxpRead,
+		UpdateContext: resourceAlkiraPeeringGatewayCxpUpdate,
+		DeleteContext: resourceAlkiraPeeringGatewayCxpDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Description: "The name of the Peering Gateway." +
+					"Once deployed, this property can not be changed.",
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Description: "Description of the CXP Peering Gateway.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"cxp": {
+				Description: "The CXP to which the Gateway is attached." +
+					"Once deployed, this property can not be changed.",
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			// TODO: change cloud_provider to be a Required value when more cloud providers are added and remove the default value.
+			"cloud_provider": {
+				Description: "The cloud provider on which the resource will be created." +
+					"Default value is AZURE and only AZURE is supported for now.",
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "AZURE",
+			},
+			"cloud_region": {
+				Description: "The region of the specified cloud provider on which the resource should be created." +
+					"E.g. if cloud_provider is AZURE, the region could be eastus.",
```

Comment:
same here, cloud_provider should be in as code block. The line seems like still too long to me. 80 break.

"The region of the specified ``cloud_provider`` on which the resource " +
"should be created. For example, if ``cloud_provider`` is ``AZURE``, the " +
"region could be ``eastus``.",

Also, notice that at the line end, you will need to leave a space there.